sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/boolAggregates.scala

```diff
@@ -56,7 +58,7 @@ case class BoolAnd(child: Expression) extends RuntimeReplaceableAggregate
   """,
   group = "agg_funcs",
   since = "3.0.0")
-case class BoolOr(child: Expression) extends RuntimeReplaceableAggregate
+case class BoolOr(child: Expression) extends ReplaceableAggregateFunction with RuntimeReplaceable
```

Comment:
At this time, shall we split this line by moving `extends ...` to next line?

Reply:
I think this is fine because it fits in one line.